.github/ISSUE_TEMPLATE/issue-default.yml

```
@@ -1,32 +1,32 @@
-name: Standard Issue Tempalte
-description: Create an Issue
-title: "[Issue]:"
-labels: ["research"]
-assignees:
-  - kitsushadow
+name: Issue
+description: Provide a title for the issue you are describing 
+title: "Please provide a clear title"
```

If we go the route of splitting up the templates, I also recommend something like this, to coincide with the corresponding type of issue being opened

```suggestion
title: "[Story]: "
```

---

Optional if using labels, but sometimes the visual prefix is nice too, since labels aren't always visible in every project view, or there might be many labels on a given story

---

Ah I see @h-m-f-t's comment about this already. Fair enough, let's use our other tools!

---

I think adding "Bug" can make sense. I think what I'm opposed to is making Random Issue Filer follow our naming conventions to get our attention. We can always edit the title to reflect that it's a bug.

---

100% agree. Naming conventions are subject to change, and too easy to mistakenly deviate from that I would not recommend filtering based on name alone. Labels are probably the most effective way to categorize in Github